<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapEditor</class>
 <widget class="QDialog" name="MapEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="mapTable">
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Row Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Search Method</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Search Term</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="addRowButton">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delRowButton">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="saveMapButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveAsButton">
      <property name="text">
       <string>Save As ...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openMapButton">
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
     <widget class="QPushButton" name="newMapButton">
      <property name="text">
       <string>New</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
